examples/module/ipddp/train_ipddp_invpend.py

```diff
@@ -0,0 +1,149 @@
+import os
+import time
+import torch
+import shutil
+import argparse
+import pypose as pp
+import pickle as pkl
+import torch.optim as optim
+from pypose.module.ipddp import IPDDP
+from tests.module.test_ipddp import InvPend
```

Comment:
Could we make a InvPend as a dynamic example in the pypose/examples, rather than in the test_ipddp? That will make our code more general and modular to be used by other users.